.github/workflows/release-brew.yaml

```diff
@@ -128,25 +128,17 @@ jobs:
         run: |
           echo "bottle_name=$(ls *.tar.gz)" >> $GITHUB_OUTPUT
 
-      - name: Get File Name
-        id: get_file_name
-        run: |
-          file_name="$(cat *.json | jq -r '."${{ env.TAP }}/${{ env.FORMULA }}".bottle.tags[].filename')"
-          echo "file_name=$file_name" >> $GITHUB_OUTPUT
-
       - name: Upload bottles as artifact
         uses: actions/upload-artifact@v4
         with:
           name: bottle-${{ matrix.os }}
           path: '*.bottle.*'
 
       - name: Upload release binary
-        uses: actions/upload-release-asset@v1.0.2
+        uses: softprops/action-gh-release@v2
         with:
-          upload_url: ${{ github.event.release.upload_url }}
-          asset_path: ${{ steps.get_package_path.outputs.bottle_name }}
-          asset_name: ${{ steps.get_file_name.outputs.file_name }}
-          asset_content_type: application/x-gzip
+          tag_name: ${{ env.RELEASE_TAG }}
+          files: ${{ steps.get_package_path.outputs.bottle_name }}
```

Comment:
I see that files is an argument to action-gh-release, but I don't see that tag_name (another argument to release) is anything other that the default github.ref and can probably be deleted.  Same also in the pypi release.

Reply:
The documentation of the action wrongly claimed it would use `github.ref` (which is fully qualified, i.e., `refs/tags/<TAG_NAME>`), but looking at the source code's action is seems it is automatically removing that `refs/tags/` prefix, so, yes, will remove `tag_name`.